src/overlays/fx/twinkle.c

```diff
@@ -1,6 +1,9 @@
 #include "common.h"
 
-#pragma GLOBAL_ASM("asm/nonmatchings/overlays/fx/twinkle/fxtwinkle_entrypoint_0.s")
+void fxtwinkle_entrypoint_0(s32 arg0)
+{
+    fxtwinkle_entrypoint_1(arg0,0x9E2);
```

Comment:
Based on the call path, this should probably be an asset enum since it gets left shifted by 2 and added to the value at D_8011B900, which is the ROM address of the assets segment.